### PR TITLE
Babylon Council #3 - Tokenomics Report

### PR DESCRIPTION
**Pull request by freakstatic**
Report generated by the Council Report Generator with the block range 1872001 - 2073601

### CODE REVIEW

**Comment by mochet** [comment]
This was approved by council: https://testnet.joystream.org/#/proposals/137

**Review by bwhm — APPROVED** [review]
LATE merge here, but I reviewed it while grading the KPIs, (just had to wait for Council approval)

**Pull request merged**
